maths/special_numbers/perfect_number.py

```diff
@@ -14,16 +14,32 @@
 
 def perfect(number: int) -> bool:
     """
+    Check if a number is a perfect number.
+
+    A perfect number is a positive integer that is equal to the sum of its proper
+    divisors (excluding itself).
+
+    Args:
+        number (int): The number to be checked.
+
+    Returns:
+        bool: True if the number is a perfect number, False otherwise.
```

Comment:
These are already in the function signature above where they are tested by `mypy` in our GitHub Actions.  We do not repeat them here because readers will get confused if one of them changes but the other does not.
```suggestion
        number: The number to be checked.

    Returns:
        True if the number is a perfect number, False otherwise.
```